lib/medusa.ex

```diff
@@ -32,6 +32,15 @@ defmodule Medusa do
 
   def start(_type, _args) do
     ensure_config_correct()
+    env = Application.get_env(:medusa, Medusa)
+    GenServer.start_link(
+      MedusaConfig, %{
+	adapter: env[:adapter],
+	message_validator: env[:message_validator]
+      },
+      [name: :config]
```

Comment:
isnt' this usually wrapped in a `start_link` inside the module, and the name of the process is `__MODULE__`?